### ISSUE TITLE
Loss interpretation

### ISSUE THREAD

**Issue by lean2206** [matterport/Mask_RCNN#2106]
Are the individual losses in percentaje? I have this doubt because my first training achieved a validation loss of 1.2 (sum of all individual validation losses) but my second training (loading the weights with 1.2 val_loss) achieved a val_loss of 0.86, but when i use the 1.2 val_loss weights for prediction, the performance is way better than the 0.86 val_loss weights. In fact, the results of the first training are the ones i was hoping to achieve.

What confuses me the most is the val_mrcnn_class_loss  and  the val_mrcnn_bbox_loss since the first training has higher values than the second one.

In the first training, after the 5th epoch i was disconnected from Colab, and in the second training after the 2nd epoch the model got overfitted

FIRST TRAINING
Epoch 5/10
110/110 [==============================] - 2754s 25s/step - 
loss: 0.3204 -
rpn_class_loss: 0.0025 - 
rpn_bbox_loss: 0.0899 - 
mrcnn_class_loss: 0.0267 - 
mrcnn_bbox_loss: 0.0574 - 
mrcnn_mask_loss: 0.1439 - 

val_loss: 1.1282 - 
val_rpn_class_loss: 3.5456e-04 - 
val_rpn_bbox_loss: 0.1737 - 
val_mrcnn_class_loss: 0.1224 - 
val_mrcnn_bbox_loss: 0.3284 - 
val_mrcnn_mask_loss: 0.5034

SECOND TRAINING

Epoch 2/10
110/110 [==============================] - 12276s 112s/step - 
loss: 0.8186 - 
rpn_class_loss: 0.0092 - 
rpn_bbox_loss: 0.2425 - 
mrcnn_class_loss: 0.0479 - 
mrcnn_bbox_loss: 0.2408 - 
mrcnn_mask_loss: 0.2781 - 

val_loss: 0.8675 - 
val_rpn_class_loss: 0.0104 - 
val_rpn_bbox_loss: 0.3051 - 
val_mrcnn_class_loss: 0.0473 - 
val_mrcnn_bbox_loss: 0.2383 - 
val_mrcnn_mask_loss: 0.2663

**Comment by SimchaGD** [comment]
Did you use the same dataset (and same train/val split) for both training?
